crontab.js

```diff
@@ -99,7 +106,7 @@ exports.set_crontab = function(env_vars, callback){
 				}
 
 				if (tab.mailing && JSON.stringify(tab.mailing) != "{}"){
-					crontab_string += "; /usr/local/bin/node " + __dirname + "/bin/crontab-ui-mailer.js " + tab._id + " " + stdout + " " + stderr;
```

Comment:
BTW nice job on the project.    I've got my docker container in production, and its working without a hitch after the latest changes.    Email was one thing I didn't test yet though.